wibb-ws-j/src/main/java/at/wibb/server/business/OfferService.java

```diff
@@ -66,10 +65,11 @@ public void addOffer(@NonNull Offer offer) {
      * @return Currently active offers.
      */
     @NonNull
-    public List<Offer> getOffers(OfferType offerType) {
-        return offerRepository.findRelevantOffers(offerType, new Date())
+    public List<Offer> getOffers(OfferFilter offerFilter) {
+        return offerRepository.findAll()
```

Comment:
in the future we can move some filtering / sorting to the persistence layer, to the mongo query